### PR TITLE
Return the CDO of the user from FSBid

### PR DESCRIPTION
**Pull request by mjoyce91**
This is not fully functioning since our mock is not updated, but the /Agents endpoint should be able to accept a `perdet_seq_num` query parameter, so that the agent of a client with a given perdet can be retrieved. This performs that lookup and adds the response to the UserProfile and Client responses.

### CODE REVIEW

**Comment by mjoyce91** [comment]
@rtirserio @scott062 - I need to check for sure that this is how the query parameter is configured in FSBid.

**Comment by mjoyce91** [comment]
@rtirserio @baldwinjordan @scott062 - would one of you be able to help me out with getting the tests to pass?

File: talentmap_api/user_profile/serializers.py [hunk 1]
```diff
@@ -49,11 +53,12 @@ class UserProfileShortSerializer(PrefetchedSerializer):
     email = serializers.CharField(source="user.email")
     initials = serializers.ReadOnlyField()
     avatar = serializers.ReadOnlyField()
+    cdo = serializers.ReadOnlyField()
     display_name = serializers.ReadOnlyField()
 
     class Meta:
         model = UserProfile
-        fields = ["username", "first_name", "last_name", "email", "phone_number", "is_cdo", "initials", "avatar", "display_name"]
+        fields = ["username", "first_name", "last_name", "email", "phone_number", "is_cdo", "initials", "avatar", "cdo", "display_name"]
```
Comment (rtirserio) [inline talentmap_api/user_profile/serializers.py]: You can't have cdo as a field in here unless its part of the model

File: talentmap_api/user_profile/serializers.py [hunk 2]
```diff
@@ -129,6 +134,12 @@ def get_favorite_positions(self, obj):
             return ({ 'id': o['id'] } for o in aps)
         return []
 
+    def get_cdo(self, obj):
+        request = self.context['request']
+        jwt = request.META['HTTP_JWT']
```
Comment (rtirserio) [inline talentmap_api/user_profile/serializers.py]: Some of the tests are failing b/c the HTTP_JWT isnt set on the request

**Comment by rtirserio** [comment]
@mjoyce91 comments on issues with tests

**Comment by mjoyce91** [comment]
Tests passing by using `cdo_info` on the UserProfile model so that old `cdo` functionality doesn't break - ready for review @rtirserio 

Also updated https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/665 to point to `cdo_info`

**Pull request merged**